.github/workflows/nightly-tests.yml

```diff
@@ -81,14 +81,14 @@ jobs:
           name: Test Reports
           path: integration/build/reports
 
-  notify-slack:
+  notify-slack-failure:
     needs: [ build, jib, test-runner ]
     runs-on: ubuntu-latest
 
     if: failure() && github.ref == 'refs/heads/main'
```

Comment:
I was going to comment that the new `if` below needed the `github.ref == 'refs/heads/main'` check, but now I think the reverse is true, that the check here isn't needed, as the only trigger for this workflow is on the `main` branch.

Reply:
Yes I think that's right. The workflow is only triggered on the `main` branch, so the condition will always be true.